bricksrc/equipment.py

```diff
@@ -126,57 +147,55 @@
             "HWS": {OWL.equivalentClass: BRICK["Hot_Water_System"], "tags": [TAG.HWS]},
         },
     },
-    "Steam_System": {"tags": [TAG.Steam, TAG.Equipment]},
+    "Water_Distribution": {
+        "tags": [TAG.Water, TAG.Distribution, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Utilize a water distribution source to represent how water is distributed across multiple destinations (pipes)"
+        ),
+    },
+    "Steam_Distribution": {
+        "tags": [TAG.Steam, TAG.Distribution, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Utilize a steam distribution source to represent how steam is distributed across multiple destinations"
+        ),
+    },
     "Solar_Panel": {"tags": [TAG.Solar, TAG.Equipment]},
-    "Shading_System": {
-        "tags": [TAG.Shade, TAG.Equipment],
-        "subclasses": {"Louver": {"tags": [TAG.Shade, TAG.Equipment, TAG.Louver]}},
+    "Louver": {"tags": [TAG.Shade, TAG.Equipment, TAG.Louver]},
+    "Lighting": {
+        "subclasses": {
+            "Luminaire": {"tags": [TAG.Luminaire, TAG.Equipment]},
+            "Luminaire_Driver": {
+                "tags": [TAG.Luminaire, TAG.Driver, TAG.Equipment],
+            },
+        },
     },
-    "Lighting_System": {
-        "tags": [TAG.Lighting, TAG.Equipment],
```

Comment:
Similar story here: we should have a `Lighting Equpiment` class to hold all of the lighting-related types